src/main/kotlin/com/target/slack/EmojiService.kt

```diff
@@ -335,18 +336,71 @@ class EmojiService(private val config: Config, private val db: Db) {
                     )
                 )
                 valid = true
+
+                val tallyResult = tallyVotes(prop)
+                // Should we report this to the admins?
+                if (tallyResult.systemReport == 0 && tallyResult.down >= config.votes.downVoteThreshold) {
```

Comment:
Its an int simply because its a tally (everything in tallyVotes is a tally).  It just so happens its unlikely to have systemReport ever tally to more than 1.